riptide_mapping/riptide_mapping2/mapping.py

```
@@ -124,18 +165,87 @@ def __init__(self):
         self.offset = Location(Point(), Vector3(), int(self.get_parameter("buffer_size").value), tuple(self.get_parameter("quantile").value))
+        self.binary_classifier = BinaryClassifier(self)
+
+        # Bin geometry shitter cursed ah subsystem
+        self.bin_fitter = BinGeometryFitter(self)
+
+        # Store seeds (which move tf and reset cov once we enter tracking)
+        self.instance1_seeded = False
+        self.instance2_seeded = False
 
         self.add_on_set_parameters_callback(self.param_callback)
         self.create_subscription(Detection3DArray, "detected_objects".format(self.get_namespace()), self.vision_callback, qos_profile_system_default)
```

Ruff (0.15.20)

[error] `.format` call has unused arguments at position(s): 0

Remove extra positional arguments at position(s): 0

(F523)